common/model/agreement.js

```diff
@@ -260,6 +261,45 @@ class agreement {
         return res.docs.map(i => Agreement.fromDocumentSnapshot(i))
       })
   }
+
+  /**
+   * Gets all the agreements that covers the current user and returns them in an array
+   * @returns {Agreement[]}
+   */
+  static getCoveredCLAs () {
+    const user = AppUser.current()
+    // Initialize to empty strings so we don't get an error when we try to toLowerCase()
+    let githubID = ''
```

Comment:
```
    let githubID = ''
    let email = ''

    return user.listAccounts().then(accounts => {
      accounts.forEach(account => {
        if (account.hostname === 'opennetworking.org') {
          email = account.email
        } else if (account.hostname === 'github.com') {
          githubID = account.username
        }
      })
    })
```
is in general dangerous to do,
accessing variables outside the function scope should be avoided, especially in asynchronous conditions as something else may change their value (eg a different promise).

Even though that shouldn't happen here I would change this code as:
 ```
 return user.listAccounts().then(accounts => {
     let values = {
       email: "",
       githubId: ""
     }
      accounts.forEach(account => {
        if (account.hostname === 'opennetworking.org') {
          values.email = account.email
        } else if (account.hostname === 'github.com') {
          values.githubID = account.username
        }
      })
      return values
    }).then(values -> {
      ....
 ```

Reply:
Makes sense! Changes should be implemented!